src/components/Calculator.js

```diff
@@ -1,42 +1,67 @@
 import React, { Component } from 'react';
+import calculate from '../logic/calculate';
 
 // eslint-disable-next-line react/prefer-stateless-function
 export default class Calculator extends Component {
+  constructor(props) {
```

Comment:
- [ ] If I try to do something like 8 % 0 or anything with % with 0, it throws an error. Please handle this
![Screenshot from 2021-11-26 20-47-38](https://user-images.githubusercontent.com/62705947/143624885-80001f21-13b0-41cd-ab0e-19ccaeb0979d.png)

Reply:
I just fixed it. Thanks for pointing it out.

Reply:
It's normal to throw an error, because % (the modulo operator) doesn't operate on zero, because we can't divide anything with 0. 
So the only fix I did is to actually display an error message in the calculator, for it not to crash.

Reply:
That is what I meant, sorry for not being clear